### PR TITLE
Example and docs for wrapping styleguide components

### PR DESCRIPTION
**Pull request by dallonf**
As discussed in #810, this seems to be the recommended approach for wrapping an existing component rather than completely replacing it.

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -392,6 +392,10 @@ module.exports = {
 
 See an example of [customized style guide](https://github.com/styleguidist/react-styleguidist/tree/master/examples/customised).
 
+If you want to wrap, rather than replace a component, make sure to import the default implementation using the full path to `react-styleguidist`. See an example of (wrapping a Styleguide component](https://github.com/styleguidist/react-styleguidist/tree/master/examples/customised/styleguide/components/Sections).
```
Comment (sapegin) [inline docs/Configuration.md]: Styleguide → Styleguidist

File: docs/Configuration.md [hunk 2]
```diff
@@ -392,6 +392,10 @@ module.exports = {
...
+
+Note that these components are not guaranteed to be safe from breaking changes in react-styleguidist updates.
```
Comment (sapegin) [inline docs/Configuration.md]: `**Note:**` to make it look nice on the site.

File: examples/customised/Readme.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # React Styleguidist customized example style guide
 
-![](https://d3vv6lp55qjaqc.cloudfront.net/items/153N2A3i2e3L2g0D043Y/Image%202017-02-11%20at%205.21.48%20PM.png)
+![](./screenshot.png)
```
Comment (sapegin) [inline examples/customised/Readme.md]: Please don't add screenshot to the repo, you can use this URL (it's also optimized):
https://d3vv6lp55qjaqc.cloudfront.net/items/0h0d3k2f172v3t3a2d1U/customised.png

**Comment by dallonf** [comment]
Thanks! Just made those tweaks

**Comment by sapegin** [comment]
CI is now failing:

> Module not found: Can't resolve 'react-styleguidist/lib/rsg-components/Sections/SectionsRenderer' in '/home/travis/build/styleguidist/react-styleguidist/examples/customised/styleguide/components'

I’d add a webpack alias, `react-styleguidist`, to customised style guide config with a comment that is’s required for test.

**Comment by dallonf** [comment]
Sorry about the messy commits - tried to fix the issue from my Windows environment using WSL. I don't have time to troubleshoot my environment right now so I'll just get back to this tomorrow when I'm at my Mac

**Comment by sapegin** [comment]
Thanks, merged!

**Pull request merged**